### PR TITLE
Upgrade dependencies

### PR DESCRIPTION
**Pull request by neilime**
Not so many fixes, only for Rxjs 6, tests are successfull

I don't have upgraded yarn.lock because I prefer not to use yarn, hope you can do it

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -65,18 +65,18 @@
...
+    "style-loader": "^0.21.0",
     "ts-helpers": "^1.1.1",
     "tsickle": ">=0.25.5",
     "tslib": "^1.9.0",
     "tslint": "^5.6.0",
     "tslint-loader": "^3.5.3",
     "typescript": "^2.5.1",
-    "webpack": "3.8.1",
-    "webpack-dev-server": "2.9.1",
+    "webpack": "4.7.0",
```
Comment (guillaume-ro-fr) [inline package.json]: > The CLI moved into a separate package: webpack-cli

Please, add `webpack-cli` dev-dependency.

File: package.json [hunk 2]
```diff
@@ -65,18 +65,18 @@
...
+    "sass-loader": "^7.0.1",
     "standard-version": "^4.2.0",
-    "style-loader": "^0.13.0",
+    "style-loader": "^0.21.0",
     "ts-helpers": "^1.1.1",
     "tsickle": ">=0.25.5",
     "tslib": "^1.9.0",
     "tslint": "^5.6.0",
     "tslint-loader": "^3.5.3",
     "typescript": "^2.5.1",
```
Comment (guillaume-ro-fr) [inline package.json]: > The Angular Compiler requires TypeScript >=2.7.2 and <2.8.0 but 2.5.3 was found instead.

You need to upgrade the `Typescript` compiler to `>= 2.7.2 || < 2.8`

**Comment by guillaume-ro-fr** [comment]
Thanks @neilime, this pull request resolve #530

**Comment by neilime** [comment]
@guillaume-ro-fr done !

**Comment by guillaume-ro-fr** [comment]
Waiting for https://github.com/neilime/ng-select/pull/1

**Comment by varnastadeus** [comment]
closing in favor #545

**Pull request closed without merging**